hack/build/build-functest.sh

```diff
@@ -0,0 +1,26 @@
+#Copyright 2018 The CDI Authors.
```

Comment:
Why is it necessary to compile the tests?  Looking that the alterations to L112 in the Makefile, they are being executed immediately afterwards, so this adds unneeded complexity.

Reply:
When I think about it again, I believe we don't need `test/functional/controller` & `test/functional/importer` we need only these under `tests/` directory.

We want to run these tests against to our own environments.

Reply:
Once #348 is merged, those test suites will be moved to their respective packages under `pkg/` with the other unit tests.  We initially wrote those as functional when we defined "functional" as being high level unit calls with complex call stacks.   There was some discussion about this and it's decided that since kubevirt's functional == e2e,  these should be considered unit tests.